modules/nf-core/plink/genome/main.nf

```diff
@@ -0,0 +1,39 @@
+process PLINK_GENOME {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/plink:1.90b6.21--h031d066_5':
+        'biocontainers/plink:1.90b6.21--h031d066_5' }"
+
+    input:
+    tuple val(meta), path(bed), path(bim), path(fam)
```

Comment:
I'm not as familiar with the tool, could you also just provide a `vcf`,`bcf` or `phe` file instead? 
If so, consider adding them as input variables. For example: 
https://github.com/nf-core/modules/blob/master/modules/nf-core/plink/epistasis/main.nf

If you can - add tests for those input files.

Reply:
Thank you, this particular plink function requires bed, bim and fam and doesn't use bcf/vcf/phe